src/components/heading/heading.stories.mdx

```diff
@@ -1,76 +1,141 @@
-import { useState } from 'react';
-import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
-import Heading from './heading.component';
-import Tile from '../tile';
-import Dl from '../definition-list/dl.component';
-import Dt from '../definition-list/dt.component';
-import Dd from '../definition-list/dd.component';
-import Link from '../link/link.component';
-import Button from '../button/button.component';
-
-<Meta title="Heading" parameters={{ info: { disable: true }}} />
+import { useState } from "react";
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
+import Heading from "./heading.component";
+import Tile from "../tile";
+import Dl from "../definition-list/dl.component";
+import Dt from "../definition-list/dt.component";
+import Dd from "../definition-list/dd.component";
+import Link from "../link/link.component";
+import Button from "../button/button.component";
+import Pill from "../pill/pill.component";
+
+<Meta title="Heading" parameters={{ info: { disable: true } }} />
 
 # Heading
+
 A standard page or section header, with a range of options, such as a help link presented as an icon, a sub-heading, a 'Back' icon, and visual dividers and separators.
 
 Make sure that the colour of all text has a contrast ratio of at least 4.5.1,
-to meet the [WCAG 2.0 AA standard](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html "WCAG 2.0 AA standard") for text less than 19px in size. 
+to meet the [WCAG 2.0 AA standard](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html "WCAG 2.0 AA standard") for text less than 19px in size.
 Webaim have a good online Contrast Checker.
 
 ## Quick Start
+
 Import `Heading` into the project.
 
 ```jsx
 import Heading from "carbon-react/lib/components/heading";
 
-const MyComponent = () => (
-  <Heading title='This is a Title' />
-);
+const MyComponent = () => <Heading title="This is a Title" />;
```

Comment:
This can be removed, we are slowly updating the component stories to just have the import path in the `Quick Start` section